src/disaster_vehicle.cpp

```diff
@@ -263,6 +263,7 @@ static bool DisasterTick_Zeppeliner(DisasterVehicle *v)
 
 		if (IsValidTile(v->tile) && IsAirportTile(v->tile)) {
 			Station *st = Station::GetByTile(v->tile);
+			st->airport.blocks.Reset(AirportBlock::Zeppeliner);
 			st->airport.blocks.Reset(AirportBlock::RunwayIn);
```

Comment:
```suggestion
			st->airport.blocks.Reset({AirportBlock::Zeppeliner, AirportBlock::RunwayIn});
```
These enum bitsets allow passing multiple enums at once.